docs/server/source/production-deployment-template-tendermint/architecture.rst

```diff
@@ -0,0 +1,210 @@
+Architecture of a BigchainDB Node
+==================================
+
+A BigchainDB Production deployment is hosted on a Kubernetes cluster and includes:
+
+* NGINX, OpenResty, BigchainDB, MongoDB and Tendermint
+  `Kubernetes Services <https://kubernetes.io/docs/concepts/services-networking/service/>`_.
+* NGINX, OpenResty, BigchainDB and MongoDB Monitoring Agent.
+  `Kubernetes Deployments <https://kubernetes.io/docs/concepts/workloads/controllers/deployment/>`_.
+* MongoDB and Tendermint `Kubernetes StatefulSet <https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/>`_.
+* Third party services like `3scale <https://3scale.net>`_,
+  `MongoDB Cloud Manager <https://cloud.mongodb.com>`_ and the
+  `Azure Operations Management Suite
+  <https://docs.microsoft.com/en-us/azure/operations-management-suite/>`_.
+
+
+.. _bigchaindb-node:
+
+BigchainDB Node
+---------------
+
+.. aafig::
```

Comment:
Maybe add a line that says something like, "To see a bigger version of the following diagram, click on it."